dev/NavigationView/NavigationView.cpp

```diff
@@ -1115,6 +1119,51 @@ void NavigationView::UpdateIsClosedCompact()
     }
 }
 
+void NavigationView::UpdatePaneButtonsWidths()
```

Comment:
I feel like this function could be refactored to use the DisplayMode condition to figure out the widths and then set those widths after. 

So instead of having:

if (DisplayModel == Minimal)
{
 if(auto const backButton = m_backButton.get())
 {
    backButton.width(width1);
 }
}
else
{
 if(auto const backButton = m_backButton.get())
 {
    backButton.width(width2);
 }
}

You could have something like:

auto backButtonWidth = []()
{
  if (DisplayModel == Minimal)
  {
    return width1;
  }
  return width2;
}();

 if(auto const backButton = m_backButton.get())
 {
    backButton.width(backButtonWidth);
 }